src/Kernel/Extension/VideoProviderMatchConstraintTest.php

```diff
@@ -99,6 +97,7 @@ public function embedCodeProvider() {
       'invalid URL' => ['https://drupal.org/project/media_entity_embeddable_video', 1],
       'invalid text' => ['I am a copy-cat and I copied this from the media entity twitter module', 1],
       'valid youtube URL' => ['https://www.youtube.com/watch?v=2YbmtzewgA4', 0],
+      'invalid youtube URL' => ['https://youtube.com/watch?v=2YbmtzewgA4', 1],
```

Comment:
I am not sure if we want to declare this invalid. It is valid URL and the fact that we currently don't handle it correctly shouldn't mean it is not.